rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -79,6 +79,50 @@ void fmt_class_string<sys_net_error>::format(std::string& out, u64 arg)
 	});
 }
 
+#ifdef _WIN32
+// Workaround function for WSAPoll not reporting failed connections
+void WSAPollEx(LPWSAPOLLFD fdArray, ULONG fds, INT timeout)
```

Comment:
Don't call it WSAPollEx, because sticking to this naming and param set only stands in the way.